README.md

```diff
@@ -1,4 +1,201 @@
+| zenodo_doi | Zenodo DOI needed for init. Leave unset to skip init. | false | unset | 
+| log_level | Log level to use, one of INFO, DEBUG, CRITICAL, ERROR, WARNING, FATAL (default INFO) | false | INFO | 
+| threshold | the minimum number of contributions required to add a user | false | 1 | 
+| force | if files exist, force overwrit | false | false |
+| ctype |Contribution type, which defaults to "code" if not set. | false | code |
+| skip_generate | skip running all-contributors generate | false | false |
+
+If you aren't familiar with all-contributors, you'll need to add some
+[commenting in your repository README](https://allcontributors.org/docs/en/cli/usage)
+so the client knows where to write.
+
+## Docker Usage
+
+If you don't want to use the GitHub Action but want to interact with the tool
+in a container, you can do that too! 
+
+### 1. Prepare the Container
+
+First, build the container from the included
+Dockerfile:
+
+```bash
+docker build -t allcontributors .
+```
+
+Then shell inside, ensuring that the entrypoint is changed to bash. You'll also
+want to bind your repository to somewhere in the container (not `/code`).
+
+```bash
+docker run -it --entrypoint bash -v $PWD/:/data allcontributors
+```
+
+In the container, a2z is installed as a python "binary" on the path:
+
+```bash
+# which a2z
+/usr/local/bin/a2z
+```
+
+The all contributors client is also on the path:
+
+```bash
+$ which cli.js
+/code/node_modules/all-contributors-cli/dist/cli.js
+```
+
+### 2. Generate .all-contributorsrc
+
+Let's now change directory to where we bound our repository as a volume.
+
+```bash
+cd /data
+```
+
+If you don't have an `.all-contributorsrc` in your present working directory (the
+root of the repository) you can use [a2z.py](a2z.py) to generate an empty one. You
+can either provide the repository name directly:
+
+```bash
+$ a2z init singularityhub/sregistry
+INFO:a2z:Generating .all-contributorsrc
+```
+
+or export it to the environment as `GITHUB_REPOSITORY` (this is how it would be discovered
+during a GitHub workflow run).
+
+```bash
+export GITHUB_REPOSITORY=singularityhub/sregistry
+$ a2z init
+INFO:a2z:Generating .all-contributorsrc
+```
+In both cases, you'll generate an empty file:
+
+```bash
+$ cat .all-contributorsrc 
+{
+    "projectName": "sregistry",
+    "projectOwner": "singularityhub",
+    "repoType": "github",
+    "repoHost": "https://github.com",
+    "files": [
+        "README.md"
+    ],
+    "imageSize": 100,
+    "commit": true,
+    "commitConvention": "none",
+    "contributors": [],
+    "contributorsPerLine": 7
+}
+```
+
+### 2. Generate .zenodo.json
+
+If you want to generate a fresh Zenodo.json, you can do that as follows:
+
+```bash
+$ a2z init --zenodo 10.5281/zenodo.1012531 singularityhub/sregistry
+```
+
+While the GitHub repository is sometimes found as a related identifier,
+this isn't always the case, so you should again provide it on the command
```

Comment:
always? but if it is associated with a github repo?  moreover `.all-contributorsrc` already contains that information too in e.g.
```
  "projectName": "ohbm2020-posters",
  "projectOwner": "datalad-datasets",
  "repoType": "github",
  "repoHost": "https://github.com",
```

Reply:
I think you are mixing up zenodo and all contributors - the statement above is referencing Zenodo. And all contributors should not be a dependency for running Zenodo alone. However, if you look at the code, you'll notice for running `general all` we _do_ take advantage of that metadata to derive the repo and don't require it for zenodo. The point above is saying "If you run this just for zenodo you should provide the repo because it's not 100% guaranteed to be served with zenodo metadata"